mne/io/egi/egimff.py

```diff
@@ -0,0 +1,411 @@
+# -*- coding: utf-8 -*-
+#
+# License: BSD (3-clause)
+
+import datetime
+import os
+import time
+
+from xml.dom.minidom import parse
+import dateutil.parser
+import numpy as np
+
+from .events import _read_events
+from .general import (_get_signalfname, _get_ep_info, _extract, _get_blocks,
+                      _get_gains)
+from ..base import BaseRaw, _check_update_montage
+from ..constants import FIFF
+from ..meas_info import _empty_info
+from ..utils import _create_chs
+from ...utils import verbose, logger, warn
+
+
+def _read_mff_header(filepath):
+    """Read mff header.
+
+    Parameters
+    ----------
+    filepath : str
+        Path to the file.
+    """
+    eeg_files, info_files = _get_signalfname(filepath, 'PNSData')
+    eeg_file = eeg_files[0]
+    fname = os.path.join(filepath, eeg_file)
+    signal_blocks = _get_blocks(fname)
+    blocknumsamps = np.sum(signal_blocks['blockNumSamps'])
+
+    pibhasref = False
```

Comment:
either make the name more explicit or add a comment, I don't know what `pib` means